parsec/api/protocol/pki.py

```diff
@@ -27,6 +27,8 @@ class PkiEnrollmentSubmitReqSchema(BaseReqSchema):
     force = fields.Boolean(required=True)
 
     submitter_der_x509_certificate = fields.Bytes(require=True)
+    # Duplicated certificate email field. (The backend need to check if the email is used without loading the certificate)
+    submitter_der_x509_certificate_email = fields.String(required=False, missing=None)
```

Comment:
can you add `allow_none=False` also ? this is not strictly needed, however I find is more explicit (and PR #2052 add `allow_none=...` everywhere `required=False`)